cookin/app/models/recipe_model.py

```diff
@@ -60,8 +61,9 @@ def total_time(self):
         return self.prep_time_minutes + self.cooking_time_minutes
 
     def relevance(self, my_tools=[]):
```

Comment:
Since the relevance function is only returning relevance in terms of tools, maybe rename the function to tool_relevance or something else that's more specific?